src/makielayout/blocks/toggle.jl

```diff
@@ -2,30 +2,39 @@ function initialize_block!(t::Toggle)
 
     topscene = t.blockscene
 
-    markersize = lift(topscene, t.layoutobservables.computedbbox) do bbox
-        min(width(bbox), height(bbox))
+    onany(topscene, t.orientation, t.length, t.markersize) do or, len, ms
+        theta = or == :horizontal ? 0 : or == :vertical ? pi/2 : or
+        t.width[] = (len - ms) * cos(theta) + ms
```

Comment:
This is not correct. The automatically determined size goes into `t.layoutobservables.autosize`. You can check against `Button` or `Label` for example. This allows the determined values to be overridden by `width` and `height` (this would not affect the size of the toggle, just the space in the layout).

Reply:
oh, thanks!  i'm still on the Makie learning curve.  fixed in the latest commit.